src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
Because this is a middleware, this can be piped with others so we should handle the request only if it makes sense. To do so we need to check against URI or Header. If this is not for us, we should quit as soon as possible. What do you think?